<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projeto Conclusão - Programação Científica Python</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cálculo de Compósitos Laminares (Orthotropic)</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>731</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Lamina</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>321</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Inputs</string>
      </property>
      <widget class="QTextEdit" name="textEdit_11">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Deformações (x,y,z) [%]</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_9">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tensões (x,y,z) [GPa]</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_8">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Angulo</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="angulo">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>E1</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="g12">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>[GPa]</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>[GPa]</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="v12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>G12</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="e1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>v12</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>E2</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="e2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>[GPa]</string>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>label_4</zorder>
     <zorder>label_6</zorder>
     <zorder>g12</zorder>
     <zorder>label_2</zorder>
     <zorder>v12</zorder>
     <zorder>label_9</zorder>
     <zorder>label_8</zorder>
     <zorder>label_5</zorder>
     <zorder>e2</zorder>
     <zorder>label_3</zorder>
    </widget>
    <widget class="QPushButton" name="calcular">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>256</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>256</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Laminado</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>e1</tabstop>
  <tabstop>e2</tabstop>
  <tabstop>v12</tabstop>
  <tabstop>g12</tabstop>
  <tabstop>angulo</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>textEdit_6</tabstop>
  <tabstop>textEdit_7</tabstop>
  <tabstop>textEdit_8</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>textEdit_10</tabstop>
  <tabstop>textEdit_9</tabstop>
  <tabstop>textEdit_11</tabstop>
  <tabstop>calcular</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
